Papr/Scenes/Search/SearchViewController.swift

```diff
@@ -66,10 +66,25 @@ class SearchViewController: UIViewController, BindableType {
             .disposed(by: disposeBag)
 
         tableView.rx.itemSelected
+            .map({ [unowned self] indexpath -> IndexPath? in
+
+                
+                if self.searchBar.isFirstResponder {
+                    self.searchBar.endEditing(true)
+                    return nil
+                } else {
+                    return indexpath
+                }
+            })
+            .unwrap()
             .map { $0.row }
             .bind(to: inputs.searchTrigger)
             .disposed(by: disposeBag)
     }
+    
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
```

Comment:
Put this function after `viewDidLoad`. I usually prefer to group all `overrides` together.